source/atlas-cli-changelog.txt

```diff
@@ -14,6 +14,48 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.12.0:
+
+{+atlas-cli+} 1.12.0
+--------------------
+
+- Drops support for Ubuntu 18.04.
+- Adds :ref:`atlas-deployments` commands so that you can work with
+  |service|, including using |fts| and {+avs+}, throughout the entire 
+  software development lifecycle, from your local environment to the 
+  cloud. To learn more, see :ref:`atlas-cli-local-cloud`.
+- Adds the following new commands to manage {+atlas-sp+} instances
```

Comment:
Suggestion: link to the Streams Processing page for more info if they're unfamiliar with it